universal-sentence-encoder/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tensorflow-models/universal-sentence-encoder",
-  "version": "0.0.4",
+  "version": "0.1.0",
```

Comment:
1) bump to 1.0, run yarn to update the lock file. Make sure the demo uses the newest version.